src/plugins/intel_cpu/src/transformations/transformation_pipeline.cpp

```diff
@@ -513,6 +514,7 @@ void Transformations::PreLpt(const std::vector<ov::element::Type>& defaultPrecis
     CPU_REGISTER_PASS_X86(manager, DecomposeIntegerDivide);
 
     CPU_REGISTER_PASS_COMMON(manager, PermuteSliceAndInterpolation);
+    CPU_REGISTER_PASS_COMMON(manager, ConvertPagedAttnInputs, config);
```

Comment:
May we move `ConvertPagedAttnInputs` before `CommonOptimizations`? Then more precise precision/shape may be got and maybe there may be new opportunities for optimization?